examples/react-native/v12/TestApp/src/components/authentication/login/LoginWithAnonymous.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {useAuth} from '@realm/react';
+
+import {Button} from '../../utility-components/Button';
+
+// :snippet-start: login-anonymous
+export const LogInWithAnonymous = () => {
+  const {logInWithAnonymous, result} = useAuth();
+
+  const performAnonymousLogin = () => {
+    logInWithAnonymous();
+  };
+
+  // Handle `result`...
+  // :remove-start:
+  return (
+    <Button
+      testID="log-in-anonymous"
+      title="Log in anonymously"
+      onPress={() => {
+        performAnonymousLogin();
+      }}
```

Comment:
```suggestion
      onPress={logInWithAnonymous}
```

This could be simplified.

Reply:
Yeah, I've been omitting the UI bits of of components for the auth stuff. Bluehawk removes everything between `:remove-start:` and `:remove-end:` so that we get samples like what's on the page: https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/DOCSP-27838/sdk/react-native/manage-users/authenticate-users/#anonymous-user

In this case, we could probably include the UI, as there's not much. But in other cases, the UI can be a lot and isn't really what we're focusing on. I think I'll keep this one as-is so it's consistent with the other examples.

I'd be happy to hear your thoughts on the balance between concise examples and "complete" examples though!

Reply:
I guess this urge to comment spawned on this block:
```
  const performAnonymousLogin = () => {
    logInWithAnonymous();
  };
```

One could simply provide `logInWithAnonymous` without wrapping it in a function.

Reply:
Oh. Yeah, that's a great point. ha! Thank you. I'll update other spots in the code that follow this pattern. Not sure why I got so wrapper happy.

Reply:
I think I initially misunderstood what you meant. Thank you for clarifying!